presto-thrift-connector/src/main/java/com/facebook/presto/connector/thrift/util/RetryDriver.java

```diff
@@ -154,4 +181,43 @@ private RuntimeException propagate(Exception e)
         throwIfUnchecked(classified);
         throw new RuntimeException(classified);
     }
+
+    private long getRetryDelayInMs(RetryStatus retryStatus)
```

Comment:
Add both of these methods to `RetryStatus`, and then I think you may be able to remove both getters.